andela/bp-esa-frontend
# #166722939 Change all freckle references to noko

#### What does this PR do?
Changes all occurrences of Noko to freckle.

#### Description of Task to be completed?
With Freckle rebranding to Noko, this task aims at renaming all occurrences of freckle to noko.

#### How should this be manually tested?
- pull in the backend PR branch(ch-freckle-noko-167193810) for freckle to noko to get the right data from the backend.
- run the project locally  using (yarn start)and visit the different pages to see the references of freckle renamed to noko



#### Any background context you want to provide?
The backend data has to be right references else you will have breaking changes.

#### What are the relevant pivotal tracker stories?
[#166722939](https://www.pivotaltracker.com/story/show/166722939)

#### Screenshots (If applicable)
